app/templates/index.hbs

```diff
@@ -11,7 +11,7 @@
   {{/if}}
   {{#if this.filteredEvents}}
     <h2 class="main-heading">{{t 'Upcoming Events'}}</h2>
-    <div class="ui stackable three column grid" id="event">
+    <div class="ui stackable three column grid" id="uimargin">
```

Comment:
Id should be unique and not repeated

Reply:
sir can you please help what type of id should i make. so that it's unique